requestor/erigolem_cli.py

```diff
@@ -1,13 +1,64 @@
 import json
-import typing
+import logging
+import time
 import urllib.parse as urlparse
+from datetime import datetime, timedelta
+from enum import Enum
+from typing import Optional, TYPE_CHECKING
```

Comment:
Just curious why `Optional` is somehow special and gets imported whether `TYPE_CHECKING`

Reply:
It's used for building models, not only type checking.